libbeat/processors/add_host_geo/add_host_geo.go

```diff
@@ -0,0 +1,59 @@
+package add_geo
+
+import (
+	"fmt"
+
+	"github.com/elastic/beats/libbeat/beat"
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/processors"
+)
+
+func init() {
+	processors.RegisterPlugin("add_host_geo", newAddGeo)
+}
+
+func newAddGeo(c *common.Config) (processors.Processor, error) {
+	config := &Config{}
+	c.Unpack(config)
+
+	ag := addGeo{
+		"name":             config.Name,
+		"location":         config.Location,
```

Comment:
Should we / can we do any validation for the location field to make sure we don't have ingestion errors?